README.md

```diff
@@ -8,16 +8,22 @@ The IBM i SQLAlchemy adapter provides a [SQLAlchemy](https://www.sqlalchemy.org/
 **Please note that this project is still in active development and is not ready for use.** :rotating_light: 
 
 ```python
-from sqlalchemy import create_engine, db
+import sqlalchemy as sa
 
+# see documentation for available connection options
+# pass connection options in url query string
+engine = sa.create_engine("ibmi://user:pass@dsn?connection_options")
```

Comment:
```suggestion
# pass connection options in url query string, eg.
# engine = sa.create_engine("ibmi://user:pass@host/?autocommit=true&timeout=10")
#
# this is a bare-minimum example which connects to *LOCAL on the given host
engine = sa.create_engine("ibmi://user:pass@host/")
```

Also add URL to SQLAlchemy docs: https://docs.sqlalchemy.org/en/13/core/engines.html#database-urls